templates/site/usage.html

```diff
@@ -0,0 +1,56 @@
+{% extends "layouts/main.html" %}
+{% set page_title = "Marble Users" %}
+{% block content %}
+    {% include "partials/banner.html" %}
+
+       {% with %}
+            {% set img_src = "images/personas/persona-mark.png" %}
+            {% set img_class = "img-fluid" %}
+            {% set img_location = "left" %}
+            {% set second_img_src = "images/personas/grad-student-secondary-image-transparentbg-640.png" %}
+            {% set second_img_class = "img-fluid" %}
+            {% set second_img_location = "right" %}
+            {% set title = "Grad Students" %}
+            {% set intro_content = "" %}
+            {% set content %}
+                <p>Marble provides an all-in-one data catalog and development environment hosted by U of T or another
+                    member of the Marble network.  Data stored on the Marble network can be accessed by all users, no
+                    matter where it is hosted.  Using web processing services tailored for climate analysis, Marble
+                    is great for those getting started in climate research. </p>
+
+                <p>Marble uses JupyterLab as its development environment.  JupyterLab lets you create self-contained
+                    notebooks that can access datasets stored on Marble's STAC catalog, and conduct data analysis using tools such as
+                    data science oriented Python packages.</p>
+
+                <p>All data storage and data processing are done on the Marble platform.  No need to
```

Comment:
Seeing as this is a headline feature, I'd move it up to the first para, potentially the second sentence in that para, rather than leave it till the end of the blurb.

Reply:
Merged  the blurb with the first paragraph.  Moved sentences in the first paragraph around to accommodate the change.